<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DockerWidget</class>
 <widget class="QWidget" name="DockerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>329</width>
    <height>110</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Docker Manager</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Version</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="combo_box_version"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Image</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="line_edit_image"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="push_button_run">
     <property name="styleSheet">
      <string notr="true">QPushButton:hover { background-color: rgb(249, 240, 107); }
QPushButton:checked { background-color: rgb(51, 209, 122); }</string>
     </property>
     <property name="text">
      <string>Run</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
